ToS RPC (#1185)
Couchers-org/couchers: app/backend/resources/terms_of_service.md

# 4. Liability and your interactions with others

Couchers.org provides a platform for users to contact one another and organise activities together. You agree to not hold us liable or responsible for user-created content or for interactions between users, including online and in person. You are solely liable and responsible for your actions on the platform, including any interactions with other users and any content that you create.

Although we are not legally responsible for user-created content, we actively moderate the platform to remove any content that is harmful, deceitful, or otherwise does not adhere to our rules. If you believe some content does not adhere to our rules, please report it to us via support@couchers.org.

When posting any image, you agree to obtain consent from any individual pictured in the image.

We are not liable if at any time, with or without prior notice, we modify, suspend or discontinue our services.

# 5. Our service providers

> We take utmost care to protect your personal data and choose our service providers carefully.

Reviewer: ```suggestion

```

Our platform is hosted on Amazon Web Services (AWS), and we use a variety of their products to provide this service to you, including hosting our servers, storing our backups, and sending transactional emails.

> We manage our mailing lists for our newsletter and general announcements using MailChimp. You agree to be added to our mailing list. You may unsubscribe at any time from within these emails. We only share minimal information with MailChimp required to send you these emails, such as your email address and display name.

Reviewer: ```suggestion
We manage a mailing list for our newsletter and general announcements using MailChimp. You agree to be added to our mailing list, and may unsubscribe at any time from within these emails. We share your email address and display name with MailChimp.
```
